test/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.2'
+
+services:    
+    testservice1:
+        image: nginx:alpine
+        labels:
+            - "traefik.frontend.rule=HostRegexp:{domain:[a-z]+};PathPrefix:/services/test1;PathPrefixStrip:/services/test1"
```

Comment:
The regex seems to disallow IP addresses, I would add this as it caused problems before.

Reply:
These services are only spawned for test execution, where traefik is not even up, but if you insist I can change that